scripts/zones/Mamook/mobs/Gulool_Ja_Ja.lua

```diff
@@ -4,31 +4,63 @@
 -----------------------------------
 
 require("scripts/globals/titles");
-
------------------------------------
--- onMobSpawn Action
------------------------------------
+require("scripts/globals/status");
+require("scripts/zones/Mamook/MobIDs");
 
 function onMobSpawn(mob)
+    mob:setLocalVar("1Hour", math.random(3,5)); 
+    mob:setMod(MOD_DOUBLE_ATTACK, 20);
 end;
 
```

Comment:
Also uses draw-in, which can be set as mobmod.  second param is 1 for single player, or 2 for alliance.  See:
https://github.com/DarkstarProject/darkstar/blob/ac0912ae626aace31fdaf0a41726a64af9570015/scripts/zones/The_Boyahda_Tree/mobs/Voluptuous_Vivian.lua#L10